test/DynamoCoreWpfTests/PackageManagerUITests.cs

```diff
@@ -291,6 +291,370 @@ public void PackageManagerConflictsWithbltinpackages()
             }
         }
 
+        [Test]
+        [Description("User tries to download a package that has a dependency that matches existing built-in package")]
+        public void PackageManagerDepMatchesBltinPackage()
```

Comment:
Added 4 new tests. All of them follow the same pattern. Two of them are for built-in packages, and the others are for non-built-in. In each set, there is one test for an exact match, and the other for a same name, different version conflict.